### PR TITLE
Don't show not active users on post reports.

### PR DESCRIPTION
**Pull request by d2bit**
[Task](https://redbooth.com/a/#!/projects/638099/tasks/18115955)

* Offer & Inquiry reports.
* Includes controller test.

### CODE REVIEW

File: app/controllers/reports_controller.rb [hunk 1]
```diff
@@ -23,7 +23,7 @@ def user_list
 
   def post_list
     @post_type = (params[:type] || "offer").capitalize.constantize
-    @posts = current_organization.posts.with_member.
+    @posts = current_organization.posts.actives.
```
Comment (rewritten) [inline app/controllers/reports_controller.rb]: Este scope `actives` me preocupa un poco, ya que en realidad hace otra join hacia `Organization`, y en otros casos hemos visto que ActiveRecord no es muy listo en entender cuando reutilizar las joins. @markets ves alguna manera de poner un scope bonito, o algo así como `organization.posts_from_active_members` para evitar dobles joins?

Por lo demás :+1: y :+1: :+1: :+1: por los tests!!!

**Comment by d2bit** [comment]
@rewritten I've created a new scope. I didn't delete the old one 'cause it's being used and I don't like to break anything..

the SQL went from:
```
=> "SELECT posts.*, members.member_uid as member_id FROM \"posts\" INNER JOIN \"organizations\" ON \"organizations\".\"id\" = \"posts\".\"organization_id\" INNER JOIN \"users\" ON \"users\".\"id\" = \"posts\".\"user_id\" INNER JOIN \"members\" ON \"members\".\"user_id\" = \"users\".\"id\" WHERE (members.organization_id = posts.organization_id) AND \"members\".\"active\" = 't' AND \"posts\".\"active\" = 't'  ORDER BY posts.updated_at DESC"
```
to:
```
=> "SELECT posts.*, members.member_uid as member_id FROM \"posts\" INNER JOIN \"members\" ON \"members\".\"id\" = \"posts\".\"user_id\" WHERE (members.active) AND (posts.active)  ORDER BY posts.updated_at DESC"
```

**Comment by rewritten** [comment]
Awesome!!!

**Comment by rewritten** [comment]
Now I should try to fix the origin of those double joins once for all.

**Pull request merged**